src/utils/url.ts

```diff
@@ -28,9 +28,19 @@ export const venueRoomUrl = (venue: WithId<AnyVenue>, roomTitle: string) => {
   return venueRoom ? venueRoom.url : venueInsideUrl(venue.id);
 };
 
-export const isExternalUrl = (url: string) =>
-  url.includes("http") &&
-  window.location.host !== new URL(getRoomUrl(url)).host;
+export const isExternalUrl = (url: string) => {
+  try {
+    const urlHost = new URL(url).host;
+    const currentHost = window.location.host;
+    return currentHost !== urlHost;
+  } catch (error) {
+    Bugsnag.notify(new Error(error), (event) => {
+      event.severity = "info";
+      event.addMetadata("utils::url::isExternalUrl", { url });
+    });
+    return false;
+  }
```

Comment:
The Bugsnag part here looks good to me!

Reply:
Added a note on https://github.com/sparkletown/sparkle/pull/919#discussion_r537446605 as to how we could be a bit more explicit in catching just the error we're expecting if you wanted to make this even more robust.